### DISCUSSION TITLE
Violet screen of Twitch at the beginning of the streams

### DISCUSSION THREAD

**Discussion by jtux270** [streamlink/streamlink-twitch-gui#810] [Q&A] [answered]
Hello, I did a quick search through this forum and the 'Issues' section, can't seem to find any possible mentions or fixes, so here it goes: at the start of almost every stream I see this [1] Twitch screen with 30 secs countdown. Not that I cannot wait a little, knowing all the limitations Twitch had already implemented for the 3rd party applications, but may be this can be bypassed somehow? Have anybody stumbled on this kind of message? Also, as an old livestreamer/streamlink user and user of this GUI, I would like to say thank you to Sebastian for his integrity and dedication through all these years.
[1]
![twitch_violet_screen](https://user-images.githubusercontent.com/5874728/125167555-ddb0b480-e1a9-11eb-955f-90026aedf44c.png)

**Comment by bastimeyer** [comment, marked as answer]
- https://github.com/streamlink/streamlink-twitch-gui/issues/754
- https://github.com/streamlink/streamlink-twitch-gui/wiki/Embedded-ads
